src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c

```diff
@@ -487,67 +481,55 @@ void EnFhgFire_Draw(Actor* thisx, GlobalContext* globalCtx) {
         gSPMatrix(gfxCtx->polyXlu.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_en_fhg_fire.c", 0x6D1),
```

Comment:
The last argument in Matrix_NewMtx is also a line number so it should be decimal ideally.